cmd/createorupdate/update.go

```diff
@@ -43,6 +52,42 @@ func update(ctx context.Context, cs *api.OpenShiftManagedCluster, p api.Plugin)
 	return nil
 }
 
+// needUpdate returns a map with the exact operations that need to be
+// executed during an upgrade. Only scale is supported for now.
+func needUpdate(cs, oldCs *api.OpenShiftManagedCluster) map[string]bool {
```

Comment:
I can't say with any certainty that I think we'll actually do this.  For now I think this PR needs to run deploy() within update() instead of outside, and it needs a hack before that which detects if the only changes are scale-related and actions them.

Reply:
Right, removed in favor of a special cased function for scaling.